chapters/hub/publish-your-pod-image.md

```diff
@@ -1,24 +1,25 @@
 
 # Publish Your Pod Image to Jina Hub
 
-You can contribute your executor into this repository and it will be automatically built into an image and published to the Jina Hub via our CI/CD pipeline.
+You can publish your custom Executor Pod to Jina Hub by contributing your Pod files to the Jina repository. You can raise a PR with all the relevant Pod files bundled as described below. Jina's CI/CD pipeline will automatically use the PR to build an image and publish it to Jina Hub.
 
-## What Files Need to be Uploaded?
```

Comment:
Put headings in sentence case (capitalize only the first word of each heading) (Unless it's a Jina term)